sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -850,3 +850,51 @@ ALTER TABLE `form_vitals` ADD `inhaled_oxygen_concentration` float(4,1) DEFAULT
 #EndIf
 
 UPDATE `list_options` SET `notes` = 'LOINC:11502-2' WHERE `list_options`.`list_id` = 'Clinical_Note_Type' AND `list_options`.`option_id` = 'laboratory_report_narrative';
+
+
+#IfMissingColumn prescriptions drug_dosage_instructions
+ALTER TABLE `prescriptions` ADD COLUMN drug_dosage_instructions longtext NULL DEFAULT NULL COMMENT 'Medication dosage instructions';
```

Comment:
make text fields bare (ok for the COMMENT, though); just trying to keep the text fields consistent since have spent resources in past standardizing these.